ontopy/graph.py

```diff
@@ -714,7 +714,7 @@ def get_edge_attrs(self, predicate, attrs):
                     self.ontology[_] for _ in relations if _ in self.ontology
                 )
                 for relation in entity.mro():
-                    if relation in rels:
+                    if relation in relations:
                         rattrs = (
                             relations[get_label(relation)]
                             if relation in rels
```

Comment:
Sure this is `relations` and not `rels`, what's the difference between these? (Defined in lines 712 and 713.)

Reply:
Okay.

This seems quite confusing to me altogether. But if it's easily readable and understandable in the code (even when coming back to this after some weeks), I don't see an issue. Otherwise, it might be time to think about how to improve this to make it more legible.

Reply:
I hope this is OK now.